### PR TITLE
Propose eip-1829 in eip-1679

### PR DESCRIPTION
**Pull request by Recmo**

### CODE REVIEW

**Comment by Recmo** [comment]
Nice, the CI found out that the link to EIP-1829 is not live yet. Blocking this PR pending #1829 merge.

File: EIPS/eip-1679.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 - Activation: TBD
 - Included EIPs: TBD
 
+## Proposed changes
```
Comment (axic) [inline EIPS/eip-1679.md]: Context: at the Eth Magicians Paris event this Monday I've proposed that #233 is extended with this rule. Proposed changes (which passed a certain level scrutiny and are well considered) can be listed in the work in progress meta.

Comment (axic) [reply EIPS/eip-1679.md]: But nobody had defined what level of scrutiny an EIP must pass before being added there.

Comment (axic) [inline EIPS/eip-1679.md]: Can you rename this to "Proposed Changes"? It seems that is the styling we are going for. Also please rebase or something to make the automerger happy.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1679 requires approval from one of (@5chdn, @axic)

**Comment by axic** [comment]
@Recmo sorry for this, but the EIP1679 changes were merged so this would need to be rebased in order to merge.

**Comment by axic** [comment]
Made the change myself.

**Pull request merged**